src/nora/components/NoraButton/NoraButton.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './NoraButton.module.scss'
+
+export const NoraButton = ({ className, children }) => {
+  return <button className={[styles.Button, className].join(' ')}>{children}</button>
```

Comment:
Can you look into adding the following props on the button?

```
      disabled={disabled}
      type={type}
      name={name}
      onClick={onClick}
      data-tid={rest['data-tid']}
      role={role}
```

Most of these should be self-evident, but `data-tid` is specifically for unit and cypress

Reply:
will do! don't know what I was thinking with this earlier smh